bin/propolis-server/src/lib/server.rs

```diff
@@ -34,6 +34,11 @@ use tokio_tungstenite::WebSocketStream;
 use crate::spec::{ServerSpecBuilder, ServerSpecBuilderError};
 use crate::vm::VmController;
 use crate::vnc::PropolisVncServer;
+use internal_dns::resolver::{ResolveError, Resolver};
+use internal_dns::ServiceName;
+pub use nexus_client::Client as NexusClient;
+use std::net::Ipv6Addr;
+use std::net::SocketAddrV6;
```

Comment:
style nit: please group these with the `std` imports in the group above.

Reply:
Done.